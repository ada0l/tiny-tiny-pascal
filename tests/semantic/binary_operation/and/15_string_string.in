var a: string; b: string;
begin writeln(a and b); end.