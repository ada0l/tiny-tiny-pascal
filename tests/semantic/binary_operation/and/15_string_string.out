(2, 17): Operator is not overloaded string and string